Quick_Sort/Randomized Quick Sort.cpp

```diff
@@ -0,0 +1,77 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+/* Partitioning function that takes last element as pivot, places 
+  the pivot element at its correct position in sorted 
+  array, and places all smaller (smaller than pivot) 
+  to left of pivot and all greater elements to right 
+  of pivot */
+
+int partition_index(int array[],int low,int high){
+    int pivot = array[high];
+    int i = low - 1;
+    for(int j = low; j < high; j++){
+        if(array[j] <= pivot){                 // If current element is smaller than or  
```

Comment:
Move the comment to next line or previous line. One line can't contain more than 80 characters including indentation and spaces.